explainaboard/feature.py

```diff
@@ -307,6 +307,8 @@ class ClassLabel:
     names: Optional[list[str]] = None
     description: Optional[str] = None
     names_file: Optional[str] = None
+    max_value: Optional[int] = None
```

Comment:
So far, we calculate them when performing `_complete_features`, and then save them in the Feature class. This should be the most effective way. 
>"or can we just take the "max" and "min" dynamically?"

Storing them in Feature class has two benefits: (1) efficiency (2) easier for different functions to use

Reply:
I see. But calculating a max is not a very slow operation, right? So it seems that (1) is not a very serious concern.

On the other hand, caching the min and max adds a significant amount of complexity to the code. If we did it in the bucketing functions it would add maybe 2 lines of code in each bucketing function, right? Instead, we've added about 50-100 lines of code in this PR just for calculating the max.